dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1824,6 +2208,33 @@
         }
       }
     },
+    "EntityWithTags": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Entity"
+        },
+        {
+          "type": "object",
+          "required": [
+            "type",
+            "tags"
+          ],
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "An entity type is the lowest (or finest) granularity at which the entity has been detected. The type maps to the specific metadata attributes associated with the entity detected."
+            },
+            "tags": {
+              "type": "array",
+              "description": "List of entity tags. Tags are to express some similarities/affinity between entities.",
+              "items": {
+                "$ref": "#/definitions/EntityTags"
+              }
+            }
+          }
+        }
+      ]
+    },
```

Comment:
Are we going to keep `EntityWithResolution`?

Reply:
Resolutions are still used by the custom entity result.